src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -695,6 +698,64 @@ class spell_mage_water_elemental_freeze : public SpellScript
     }
 };
 
+/* 112965 - Finger of Frost
+   228597 - Frostbolt
+   84721  - Frozen Orb */
+class spell_mage_finger_of_frost : public SpellScriptLoader
+{
+public:
+    spell_mage_finger_of_frost() : SpellScriptLoader("spell_mage_finger_of_frost") { }
+
+    class spell_mage_finger_of_frost_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_mage_finger_of_frost_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_MAGE_FINGERS_OF_FROST });
+        }
+
+        void HandleAfterCast()
+        {
+            Unit* caster = GetCaster();
+            uint32 spellId = GetSpell()->GetSpellInfo()->Id;
+            uint32 chance = 0;
+
+            SpellInfo const* fingerFrostInfo = sSpellMgr->GetSpellInfo(SPELL_MAGE_FINGERS_OF_FROST, GetCastDifficulty());
```

Comment:
```cpp
TrinityCore/TrinityCore/src/server/scripts/Spells/spell_mage.cpp:724:30: fatal error: 
      unused variable 'fingerFrostInfo' [-Wunused-variable]
            SpellInfo const* fingerFrostInfo = sSpellMgr->GetSpellInfo(S...
                             ^
```
Please note that the build checks stop looking for more build warnings (treated as error) once one has been found.
```suggestion
            SpellInfo const* /*fingerFrostInfo*/ = sSpellMgr->GetSpellInfo(SPELL_MAGE_FINGERS_OF_FROST, GetCastDifficulty());
```